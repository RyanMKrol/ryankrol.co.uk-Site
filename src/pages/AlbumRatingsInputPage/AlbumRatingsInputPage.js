import React from 'react';

import { AlbumRatingsForm } from '../../components';

import styles from './AlbumRatingsInputPage.module.css';

/**
 * Page introducing the site
 *
 * @returns {string} JSX Content
 */
function AlbumRatingsInputPage() {
  return (
    <div className={styles.container}>
      <h1 className="light">Rate an album!</h1>
      <AlbumRatingsForm />
    </div>
  );
}

export default AlbumRatingsInputPage;
