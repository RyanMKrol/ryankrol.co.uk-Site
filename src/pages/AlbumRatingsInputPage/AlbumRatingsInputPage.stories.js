import React from 'react';

import AlbumRatingsInputPage from './AlbumRatingsInputPage';

export default {
  title: 'Pages/AlbumRatingsInputPage',
};

export const Default = () => <AlbumRatingsInputPage />;
