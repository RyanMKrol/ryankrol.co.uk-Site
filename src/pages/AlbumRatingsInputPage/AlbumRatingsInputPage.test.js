import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import AlbumRatingsInputPage from './AlbumRatingsInputPage';

it('renders AlbumRatingsInputPage correctly', () => {
  const header = shallow(<AlbumRatingsInputPage />);
  expect(toJson(header)).toMatchSnapshot();
});
