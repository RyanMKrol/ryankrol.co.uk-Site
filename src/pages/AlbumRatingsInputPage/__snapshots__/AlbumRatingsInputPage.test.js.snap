// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AlbumRatingsInputPage correctly 1`] = `
<div
  className="container"
>
  <h1
    className="light"
  >
    Rate an album!
  </h1>
  <AlbumRatingsForm />
</div>
`;
